rfc/helm-import.md

```diff
@@ -0,0 +1,515 @@
+# Helm Import 
+
+* **Original Author(s):** @vinayak-kukreja
+* **Tracking Issue:** https://github.com/cdk8s-team/cdk8s/issues/1271
+* **API Bar Raiser:** @iliapolo
+
+Users are now able to import helm charts into their cdk8s app with `cdk8s import` command.
+
+## Working Backwards
+
+### README
+
+You can add a helm chart to your cdk8s application by running:
+
+```
+cdk8s import helm:https://airflow.apache.org/airflow@1.8.0
+```
+
+Here, `airflow` is the helm chart and `1.8.0` is the chart version.
+
+Then you can use this chart within your application:
+
+```typescript
+import { Construct } from 'constructs';
+import { App, Chart, ChartProps } from 'cdk8s';
+import { Airflow } from './imports/airflow';
+
+export class MyChart extends Chart {
+  constructor(scope: Construct, id: string, props: ChartProps = { }) {
+    super(scope, id, props);
+
+    new Airflow(this, 'airflow', {
+      values: {
+        fullnameOverride: 'Foo',
+      }
+    });
+  }
+}
+
+const app = new App();
+new MyChart(app, 'typescript-demo');
+app.synth();
+```
+
+> **Note:**
+>
+> * You would need `Helm` to be installed on your machine for using this feature. 
+> * We assume that you are authenticated to access the helm chart being mentioned in the url.
+> * If a helm chart contains a `values.schema.json` then the `values` properties within the construct properties for the chart would have specific types. For instance, in the prior example `fullnameOverride` is not of `any` type and instead is of `string` type since `airflow` chart has a `values.schema.json` file in it.
+
```

Comment:
This should be front and center - not as a note. Also lets make sure the example chart we use actually does have a `values.schema.json` file and mention that if *your* chart doesn't, the values won't be typed.

Reply:
I see. `airflow` does have a schema file. Are you suggesting that we emit a warning to the user if the chart they are using does not have a schema file and let them know that the value property would not be typed?

Reply:
No I don't mean with a warning - just in the documentation.